### PR TITLE
Add name attribute to user

### PR DESCRIPTION
**Pull request by biswajit-hypto**
`name` attribute is added to `User`

### CODE REVIEW

File: iam_openapi_spec.yml [hunk 1]
```diff
@@ -1805,9 +1805,11 @@ components:
         description: Brief description about ACME
         rootUser:
           username: username
+          name: Lorem Ipsum
```
Comment (karthick-vinod) [inline iam_openapi_spec.yml:1808]: Should be marked as required.

File: src/main/kotlin/com/hypto/iam/server/validators/RequestModelValidator.kt [hunk 2]
```diff
@@ -188,6 +188,11 @@ val orgNameCheck = Validation<String> {
     pattern(ORGANIZATION_NAME_REGEX) hint ORGANIZATION_NAME_REGEX_HINT
 }
 
+val nameOfUserCheck = Validation<String> {
```
Comment (karthick-vinod) [inline src/main/kotlin/com/hypto/iam/server/validators/RequestModelValidator.kt:191]: Do we need to add checks like
1. Min chars
2. Char count without spaces etc.?

File: iam_openapi_spec.yml (hunk 1 above)
Comment (biswajit-hypto) [reply iam_openapi_spec.yml:1808]: It's required already.
https://github.com/hwslabs/iam/blob/f47370cd368a5603cc62ca646decc4382ea24f48/iam_openapi_spec.yml#L2194-L2208

File: src/main/kotlin/com/hypto/iam/server/validators/RequestModelValidator.kt (hunk 2 above)
Comment (biswajit-hypto) [reply src/main/kotlin/com/hypto/iam/server/validators/RequestModelValidator.kt:192]: Done

File: src/main/kotlin/com/hypto/iam/server/extensions/Validators.kt [hunk 3]
```diff
@@ -35,9 +35,15 @@ fun ValidationBuilder<String>.dateTime(
     try {
         val dateTime = LocalDateTime.parse(it, format)
         when (nature) {
-            TimeNature.ANY -> { true }
-            TimeNature.PAST -> { dateTime.isBefore(LocalDateTime.now()) }
-            TimeNature.FUTURE -> { dateTime.isAfter(LocalDateTime.now()) }
+            TimeNature.ANY -> {
```
Comment (karthick-vinod) [inline src/main/kotlin/com/hypto/iam/server/extensions/Validators.kt:38]: nit: Avoid unnecessary formatting changes

**Review by karthick-vinod — APPROVED** [review]

File: src/main/kotlin/com/hypto/iam/server/extensions/Validators.kt (hunk 3 above)
Comment (karthick-vinod) [reply src/main/kotlin/com/hypto/iam/server/extensions/Validators.kt:38]: Ignore if ktlint does it
